docker-compose.yml

```diff
@@ -1,6 +1,15 @@
 version: '3.4'
 
 services:
+  planet-learning-database:
```

Comment:
The database could be persistent in order to keep the data safe in case of a container crash

Reply:
I'll make a separate PR for this, this PR is big enough as it is